tests/relay_integration/lang/javascript/test_plugin.py

```diff
@@ -427,9 +442,13 @@ def test_sourcemap_source_expansion(self):
 
         event = self.post_and_retrieve_event(data)
 
-        assert event.data["errors"] == [
-            {"type": "js_no_source", "url": "http//example.com/index.html"}
-        ]
+        # FIXME: Unify this assertion once we add error mapping.
+        if process_with_symbolicator:
+            assert "errors" not in event.data
+        else:
```

Comment:
It actually won't! :D Because we don't map errors in `merge_frame` yet, so there will be no failures, and we can update asserting in the same PR as we introduce mappings.

Reply:
:-D haha well thats a lucky coincidence :-D